Formula/k/kp-diff.rb

```diff
@@ -0,0 +1,54 @@
+class KpDiff < Formula
+  desc "CLI utility to diff KeePass database"
+  homepage "https://github.com/aivanovski/kp-diff"
+  url "https://github.com/aivanovski/kp-diff/releases/download/0.5.1/kp-diff.jar"
+  sha256 "e0c7d7b1bf6895a7f5cfb976fb6a33034b0ac5554b1ce21a4db6ede70c4556df"
+  license "Apache-2.0"
+
+  depends_on "openjdk@11"
```

Comment:
Why not the more modern jdk?

Reply:
I'm actually not sure what version of jdk use here. But as I know, usually java application are bundled with specific jre, what guarantees that the application will work correctly.
This application was compiled with jdk 11 and it was tested with it, that's why jdk 11 was chosen.

Reply:
Homebrew requires everything use the latest version of dependencies unless that version is known to be broken, so this should be:
```suggestion
  depends_on "openjdk"
```

Reply:
Fixed